Lib9c/Model/Character/Player.cs

```diff
@@ -60,6 +60,10 @@ public object Clone()
         public CollectionMap monsterMap;
         public CollectionMap eventMap;
 
+        // ??
```

Comment:
Could you fill in the empty part of the comment?

Reply:
Oh! I missed it. Thanks.

Reply:
@sonohoshi I fixed it.